tests/iostability/test_iostability_happy_path.py

```diff
@@ -195,3 +196,114 @@ def test_disk_near_full_s3bench(self):
         self.test_completed = True
         self.log.info("ENDED: Perform disk storage near full once and read in loop for %s days",
                       self.duration_in_days)
+
+    @pytest.mark.lc
+    @pytest.mark.io_stability
+    @pytest.mark.tags("TEST-40042")
+    def test_iteration_write_read_partial_delete(self):
+        """Perform iterations of 30% writes of user capacity ,
```

Comment:
Can you divide into more reusable functions and use them in similar tests in this module?

Reply:
Added two function in lib.
perform_write_to_fill_system_percent and delete_workload